.github/workflows/build-test-release.yml

```diff
@@ -33,19 +33,29 @@ jobs:
 
   pre-commit:
     runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [ 3.7, 3.8, 3.9, 3.10, 3.11 ]
+
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v4
         with:
-          python-version: "3.7"
+          python-version: "${{ matrix.python-version }}"
       - uses: pre-commit/action@v3.0.0
   test-unit:
     runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [ 3.7, 3.8, 3.9, 3.10, 3.11 ]
```

Comment:
You need to quote "3.10" and "3.11", otherwise GitHub does not recognise it as "3.10", but as "3.1".

Reply:
Thanks, I didn't know that issue